specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -6208,6 +6208,14 @@
       },
       "description": "Specifies the boot diagnostic settings state. <br><br>Minimum api-version: 2015-06-15."
     },
+    "BillingProfile": {
+      "maxPrice": {
```

Comment:
move this property to inside of "properties":

Reply:
Thank you for pointing out. This was causing the error initially. Made the change.